packages/office-ui-fabric-react/src/components/Autofill/Autofill.tsx

```diff
@@ -64,16 +64,19 @@ export class Autofill extends BaseComponent<IAutofillProps, IAutofillState> impl
 
   // tslint:disable-next-line function-name
   public UNSAFE_componentWillReceiveProps(nextProps: IAutofillProps): void {
-    let newValue;
-
     if (this.props.updateValueInWillReceiveProps) {
-      newValue = this.props.updateValueInWillReceiveProps();
+      const updatedInputValue = this.props.updateValueInWillReceiveProps();
+      // Don't update if we have a null value or the value isn't changing
```

Comment:
What about undefined?

Reply:
updateValueInWillReceiveProps is not [typed](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Autofill/Autofill.types.ts#L82-L90) as returning undefined. I can certainly add a check but to me the type is pretty explicit about what can be returned.